incubator/hnc/internal/reconcilers/object.go

```diff
@@ -699,6 +705,21 @@ func (r *ObjectReconciler) shouldPropagateSource(log logr.Logger, inst *unstruct
 	}
 }
 
+// getNoneSelector returns true indicates that user do not want this object to be propagated
+func (r *ObjectReconciler) getNoneSelector(log logr.Logger, inst *unstructured.Unstructured) bool {
+	annot := inst.GetAnnotations()
+	noneSelectorStr, ok := annot[api.AnnotationNoneSelector]
+	if !ok {
+		return false
+	}
+	noneSelector, err := strconv.ParseBool(noneSelectorStr)
+	if err != nil {
+		log.Error(err, "Invalid noneSelector value, it should be either true or false.")
```

Comment:
BTW should this error be surfaced? Seems yes? If so, maybe add the same todo comment?

Reply:
done